src/ctree-app/ctree-landing.html

```diff
@@ -27,41 +27,83 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html" />
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-drawer/app-drawer.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-scroll-effects/app-scroll-effects.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-header/app-header.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-header-layout/app-header-layout.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-toolbar/app-toolbar.html"
+/>
+<link rel="import" href="../../bower_components/iron-list/iron-list.html" />
+<link
+  rel="import"
+  href="../../bower_components/iron-location/iron-location.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/iron-location/iron-query-params.html"
+/>
+<link rel="import" href="../../bower_components/iron-meta/iron-meta.html" />
+<link
+  rel="import"
+  href="../../bower_components/iron-scroll-target-behavior/iron-scroll-target-behavior.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/iron-selector/iron-selector.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/paper-button/paper-button.html"
+/>
+<link rel="import" href="../../bower_components/paper-fab/paper-fab.html" />
+<link
+  rel="import"
+  href="../../bower_components/paper-icon-button/paper-icon-button.html"
+/>
+<link rel="import" href="../../bower_components/paper-input/paper-input.html" />
+<link
+  rel="import"
+  href="../../bower_components/paper-spinner/paper-spinner.html"
+/>
+<link rel="import" href="../../bower_components/paper-tabs/paper-tabs.html" />
+<link
+  rel="import"
+  href="../../bower_components/paper-tooltip/paper-tooltip.html"
+/>
 
-<link rel="import" href="../ctree-dialogs/ctree-element-dialog-accessor.html">
-<link rel="import" href="../ctree-dialogs/ctree-new-element-dialog-accessor.html">
-<link rel="import" href="../ctree-dialogs/ctree-user-dialog-accessor.html">
-<link rel="import" href="../ctree-import/ctree-import.html">
-<link rel="import" href="../ctree-toggle-button/ctree-toggle-button.html">
+<link rel="import" href="../ctree-dialogs/ctree-element-dialog-accessor.html" />
+<link
+  rel="import"
+  href="../ctree-dialogs/ctree-new-element-dialog-accessor.html"
+/>
+<link rel="import" href="../ctree-dialogs/ctree-user-dialog-accessor.html" />
+<link rel="import" href="../ctree-import/ctree-import.html" />
+<link rel="import" href="../ctree-toggle-button/ctree-toggle-button.html" />
 
-<link rel="import" href="ctree-app-icon.html">
-<link rel="import" href="icons.html">
+<link rel="import" href="ctree-app-icon.html" />
+<link rel="import" href="icons.html" />
 
 <dom-module id="ctree-landing">
   <template>
     <style>
       :host {
         display: block;
-        --app-primary-color: #5F9F00;
+        --app-primary-color: #5f9f00;
```

Comment:
Probably not worth changing this.  It's best to try to avoid changing values which don't provide value.